lib/pymedphys/cli/dev.py

```diff
@@ -15,6 +15,7 @@ def add_docs_parser(dev_subparsers):
     parser = dev_subparsers.add_parser("docs")
 
     parser.add_argument("--live", help="Make the docs live reload", action="store_true")
+    parser.add_argument("--output", help="Custom output directory")
```

Comment:
will this be obvious to the user that it is the custom output directory of the docs? (vs. something else)

Reply:
Yup, I can see how that could be ambiguous. Just made a small addition. I might need to jump on a PC and re-blackify it though.